### PR TITLE
creating AVD article

### PR DESCRIPTION
**Pull request by NickIliev**
related to [https://github.com/NativeScript/docs/issues/358](https://github.com/NativeScript/docs/issues/358)

### CODE REVIEW

File: tooling/android-virtual-devices.md [hunk 1]
```diff
@@ -0,0 +1,864 @@
...
+
+## Create a new emulator 
+
+1. Open **Android SDK Manager** 
+
+    It can be started from command prompt / shell with command `android`
+
+2. Make sure that the system images for the AVDs are installed. 
+
+    In the screenshot below are shown installed system images for Android API 24. If additional system images are needed (e.g. for different API level) select them and press **Install Packages** button at the bottom right corner.
```
Comment (ErjanGavalji) [inline tooling/android-virtual-devices.md]: I'd reword this as "The screenshow below shows the Android API 24 system images installed on the computer..."

File: tooling/android-virtual-devices.md [hunk 2]
```diff
@@ -0,0 +1,864 @@
...
+
+    ![Installed AVDs for API 24 in Andrid SDK](../img/create-avd/avd-001.png "Installed AVDs for API 24 in Andrid SDK")
+
+    You can accelerate the virtual device using Virtualization Technology. For hardware with Intel CPUs this can be achieved with HAXM installer.
+
+    ![Enabling HAXM](../img/create-avd/avd-002.png "Enabling HAXM")
+
+    Once your HAXM is downloaded by the Android SDK Manager navigate to the source folder (usually in `<sdk_path>\extras\intel\Hardware_Accelerated_Execution_Manager\`) and manually install **Intel HAXM** (intelhaxm-android.exe). For additional information on HAXM follow this [link](https://software.intel.com/en-us/android/articles/speeding-up-the-android-emulator-on-intel-architecture)
```
Comment (ErjanGavalji) [inline tooling/android-virtual-devices.md]: "Once the SDK Manager downloads the HAXM installer, navigate to the..."

File: tooling/android-virtual-devices.md [hunk 3]
```diff
@@ -0,0 +1,864 @@
...
+
+The example above will start the application on device with id 2 (sdk_phone_x86_64). The same rule applies for `livesync` command
+
+
+To launch a NativeScript application on all connected devices simultaneously, simply ignore the `--device <id>` flag.
+
+```
+tns run android
+```
+
```
Comment (ErjanGavalji) [inline tooling/android-virtual-devices.md:106]: Why so many empty lines here?

**Review by ErjanGavalji — COMMENTED** [review]
Several suggestions...

File: tooling/android-virtual-devices.md (hunk 3 above)
Comment (NickIliev) [reply tooling/android-virtual-devices.md:106]: All revised and refactored!

**Review by ErjanGavalji — APPROVED** [review]
Thanks @NickIliev
Looks good to me!
